manifest.php

```diff
@@ -28,7 +28,7 @@
     'name'        => 'taoQtiItem',
     'label'       => 'QTI item model',
     'license'     => 'GPL-2.0',
-    'version'     => '12.3.0',
+    'version'     => '12.4.0',
```

Comment:
Since there is a dependency on a feature brought by taoCore, you should also update the required version of `tao`

Reply:
resolved